### DISCUSSION TITLE
Transfer of repo between personal accounts

### DISCUSSION THREAD

**Discussion by iyhasan** [community/community#78339] [New to GitHub]
I initiated a transfer of my repo from account A to B, but I didn't get any emails on account B about completing the transfer, am I missing something?

**Comment by erickdc7** [comment]
When transferring a repository from one personal account to another, the new owner should receive a confirmation email with instructions for accepting the transfer. If you haven't received the email on account B, here are a few steps you can take:

1. **Check Spam/Junk Folder**: Sometimes, the confirmation email might be filtered into the spam or junk folder. Make sure to check these folders in account B's email.
2. **Email Settings**: Verify that the email address associated with account B is correct and is capable of receiving emails.
3. **Resend the Invitation**: If the invitation has expired (it expires after one day), you may need to initiate the transfer again from account A.
4. **GitHub Notifications**: Check the notification settings in account B to ensure that emails from GitHub are not being blocked or filtered out.
5. **Contact GitHub Support**: If you've checked all of the above and still haven't received the email, you may need to contact GitHub support for further assistance.

For more detailed information on repository transfers and troubleshooting, you can refer to the [GitHub Docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository). Remember, the transfer must be accepted within one day, or the invitation will expire.
